target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -28,6 +28,10 @@ buffalo,whr-g301n)
 	ucidef_set_led_switch "lan3" "LAN3" "$boardname:green:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "$boardname:green:lan4" "switch0" "0x10"
 	;;
+dlink,dir-859-a1)
+	ucidef_set_led_switch "internet" "WAN" "d-link:green:internet" "switch0" "0x20"
+	ucidef_set_led_wlan "wlan" "WLAN" "d-link:green:wlan" "phy0tpt"
```

Comment:
Please set the trigger in the dts file. You should find quite some examples in ath79/dts/.

Reply:
I think so.